### ISSUE TITLE
calcite-stepper throws __spreadArrays error

### ISSUE THREAD

**Issue by drewctate** [Esri/calcite-components#649]
Using the `calcite-stepper` component as follows:

```
        <calcite-stepper numbered scale="m">
            <calcite-stepper-item scale="m" item-title="title 1" />
            <calcite-stepper-item scale="m" item-title="title 2" />
            <calcite-stepper-item scale="m" item-title="title 3" />
        </calcite-stepper>
```
I see the following error:

<img width="689" alt="Screen Shot 2020-06-08 at 10 43 51 AM" src="https://user-images.githubusercontent.com/315764/84057609-24aff300-a975-11ea-846c-fd4d1be4fc58.png">

**Comment by macandcheese** [comment]
Thanks for reporting @drewctate - is this in a Vue app by any chance? Similar: https://github.com/Esri/calcite-components/issues/531

**Comment by drewctate** [comment]
Hi @macandcheese. I did some sleuthing.

### TL;DR and suggested fix
The destructuring operator that is used on a `Set` in the `sortItems` function [here](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-stepper/calcite-stepper.tsx#L287) breaks in `esm-es5`-transpiled builds. A quick fix would be to avoid using the spread operator on a Set anywhere in `calcite-components`.

This doesn't appear to depend on the javascript framework (though Hub uses Ember).

### Long story:
- run `npm build`
- look at `./dist/esm-es5/calcite-stepper_2.entry.js`, line 38 (pasted below for convenience)

```
for (var s = 0, i = 0, il = arguments.length; i < il; i++) s += arguments[i].length;
```

Notice that `s` is initialized to zero, and then `arguments[i].length` is added thereto. Well, looking at the invocation on line 265 of the same file:

```
__spreadArrays(new Set(items));
```

you'll see that `arguments[0]` is a `Set`. The problem is that a `Set` has no `length` property. When `set.length` is added to `s`, the result is `NaN` which messes everything up.

**Comment by macandcheese** [comment]
Ah, nice. Explains why it worked with CJS loader in linked issue. Thanks for 🕵️!
